flexmeasures/ui/templates/crud/asset.html

```diff
@@ -1421,51 +1421,51 @@ <h5 class="card-title">${sensor.name}</h5>
         const alertStyle = "alert alert-light";
         if (selectedOption === "consumption-price") {
             const description = "Set the sensor that represents the consumption price of the site. This value will be used in the optimization";
-            const allowedUnits = ["EUR", "EUR/MWh"];
+            const allowedUnits = ["EUR/MWh", "JPY/MWh", "USD/MWh, and other currencies."];
```

Comment:
Please change all mentions of JPY/MWh to JPY/kWh. Two reasons:
- Japanese wholesale market prices are more commonly stated per kWh.
- It demonstrates that we support multiple options for the denominator, too.